terminus-ui/tsconfig.lint.json

```diff
@@ -1,10 +1,8 @@
 {
-  "extends": "./../tsconfig.json",
+  "extends": "./tsconfig.json",
```

Comment:
it looks like this is a new file, but the whole thing isn't highlighted. 

Are these other excluded items excluded in tsconfig.json? I don't see that file in this PR.

Reply:
That file has always existed and has no changes. Before we were extending like `this file < file B < file A` and now we are just extending straight from `file A`.